### DISCUSSION TITLE
hiding the "create" button in the header after creating the related post

### DISCUSSION THREAD

**Discussion by Kenny201** [filamentphp/filament#7874] [Help] [answered]
There is an output of a table of related records in the resource under the fields of the parent element. There is a "create" button to add a related entry to this item. Please tell me whether it is possible to implement hiding this button after creating a related record so that it is impossible to add another record, which should not be logically

**Comment by danharrin** [comment]
You can pass a boolean to hidden() based on if `$this->record` has a related record or not

**Comment by Z3d0X** [comment, marked as answer]
```php
Tables\Actions\CreateAction::make()
    ->hidden(fn (RelationManager $livewire) => $livewire->getRelationship()->exists()),
```

**Reply by Kenny201** [reply]
Cool!!! Thanks you)))
